app/scenes/RootScreen/reducer.js

```diff
@@ -15,8 +16,8 @@ export const initialState = fromJS({
 
 export const rootContainerReducer = (state = initialState, action) =>
   produce(state, () => {
-    switch (action.type) {
-      default:
-        return state;
-    }
+    const stateReturn = {
```

Comment:
this introduces more complexity than the switch. Let's continue using the switch please